viabel/tests/test_convenience.py

```diff
@@ -1,7 +1,7 @@
-import autograd.numpy as anp
+import jax.numpy as jnp
```

Comment:
why do you need to use Jax for everything here? seems only necessary if you are going to autodiff. Same question applies to other test files